templates/items/parts/item-description2.hbs

```diff
@@ -0,0 +1,95 @@
+{{#*inline "description"}}
+    <div class="card description {{#if collapsible}}collapsible{{/if}} {{#if (lookup @root.collapsed target)}}collapsed{{/if}}"
+         data-target="{{ target }}">
+        <div class="header">
+            <span>{{ localize label }}</span>
+            {{#if @root.owner}}
+            <button type="button" class="description-edit unbutton" data-target="{{ target }}"
+                    aria-label="{{ localize "DND5E.DescriptionEdit" description=(localize label) }}">
+                <i class="fas fa-feather"></i>
+            </button>
+            {{/if}}
+        </div>
+        <div class="details collapsible-content">
+            <div class="wrapper">
+                {{ editor enriched target=target button=false editable=false engine="prosemirror" }}
+            </div>
+        </div>
+    </div>
+{{/inline}}
+
+<div class="tab {{#if singleDescription}}single-description{{/if}}" data-group="primary" data-tab="description">
+
+    {{#if editingDescriptionTarget}}
+        {{ editor enriched.editing target=editingDescriptionTarget button=false editable=true engine="prosemirror"
```

Comment:
Any reason for favoring this over the new `<prose-mirror>` element?

Reply:
We have an `activateEditor` override in `ItemSheet` that configures the menu plugin to set `editingDescriptionTarget = null` when the editor is saved. I couldn't figure out how to replicate that behaviour with the `<prose-mirror>` element (admittedly I didn't spend too much time on it). Listening to the `change` event almost works, but it only fires if the editor content actually changes. We could subclass the element or add mutation observers maybe, but I would rather just improve the element in core to fire a `close` event or something.